core/src/main/scala/org/bitcoins/core/script/constant/BytesToPushOntoStack.scala

```diff
@@ -25,7 +25,7 @@ object BytesToPushOntoStack
     override val opCode = num
   }
 
-  override def operations: Seq[BytesToPushOntoStack] =
+  override val operations: Seq[BytesToPushOntoStack] =
```

Comment:
this is a quick optimization so we don't re-create `BytesToPushOntoStack` everytime we parse a script.

Reply:
nah, it doesn't really matter `lazy` or `val`, the problem is it being a `def` so it has to be recreated _everytime_